proto/attestation/expected_value.proto

```diff
@@ -0,0 +1,42 @@
+
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.attestation.v1;
+
+import "proto/digest.proto";
+
+option go_package = "proto/oak/attestation/v1";
+option java_multiple_files = true;
+option java_package = "com.google.oak.attestation.v1";
+
+message VerificationSkipped {}
```

Comment:
This is how I avoided the name collision. The rationale for the name here is that this isn't an "instruction" to skip verification, but rather than acknowledge of an earlier instruction to skip verification.

Reply:
Another strategy for the name collision to consider is to further divide the packages. i.e.:

oak.attestation.v1.reference
oak.attestation.v1.expectation
oak.attestation.v1.endorsement

...etc.

Reply:
I don't have a strong preference, but I think further dividing the packages will involve a lot more work. So unless there is a strong motivation to do that I would go with the easier route of just using a different message name.